src/pieces/logs/serverLog.ts

```diff
@@ -309,6 +310,41 @@ async function processMessageDelete(msg: Message | PartialMessage, serverLog: Te
 	}
 
 	serverLog.send({ embeds: [embed], files: attachments });
+
+	const bot = msg.client;
+	handleExpDetract(bot, msg);
```

Comment:
We should probably only detract xp when it's not a sage command. You don't get xp for using sage commands in the first place, so probably check if `msg.content.startsWith('s;')`

Reply:
Good call!